lib/index.mjs

```diff
@@ -0,0 +1,33 @@
+import checkAgainstRule from './utils/checkAgainstRule.mjs';
+import createPlugin from './createPlugin.mjs';
+import createStylelint from './createStylelint.mjs';
+import formatters from './formatters/index.mjs';
+import { longhandSubPropertiesOfShorthandProperties } from './reference/properties.mjs';
+import postcssPlugin from './postcssPlugin.mjs';
+import report from './utils/report.mjs';
+import resolveConfig from './resolveConfig.mjs';
+import ruleMessages from './utils/ruleMessages.mjs';
+import rules from './rules/index.mjs';
+import standalone from './standalone.mjs';
+import validateOptions from './utils/validateOptions.mjs';
```

Comment:
minor

Maybe having a `./utils/index.mjs` would be useful so that we could have one import for utils instead of 4.

Reply:
I have no strong objection, but `utils/index.mjs` (or `utils.mjs`) would bring two patterns into the codebase, for example:

```js
import ruleMessages from "./utils/ruleMessages.mjs"
import validateOptions from "./utils/validateOptions.mjs"

// vs

import { ruleMessages, validateOptions } from "./utils/index.mjs"
```

This might spoil consistency. And we would need to maintain the new file (`utils/index.mjs`) additionally.

If we'd perform the refactoring, I recommend it after this migration, at least.